packages/core/src/Stores/client-store.js

```diff
@@ -621,7 +621,7 @@ export default class ClientStore extends BaseStore {
 
     @computed
     get is_options_blocked() {
-        return isOptionsBlocked(this.residence);
+        return this.residence ? isOptionsBlocked(this.residence) : true;
```

Comment:
If the client is logged out, we can consider `clients_country` in  `website_status` which will be based on his IP address.
```
return isOptionsBlocked(this.residence || this.website_status.clients_country);
```